### PR TITLE
selector specificity

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

**Review by AbbasShaikh0304 — APPROVED** [review]

**Review by faran4engg — CHANGES_REQUESTED** [review]
correct folder name 😒

File: tasks-by-users/Naufil-Shemle/selector-specificity/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    <div id="main-content">
+      <p>Hello there <strong>ninjas!</strong> contact us below.</p>
+      <p class="test">Aren't paragraph's cool?</p>
+    </div>
+
+    <p>None contaoned P tag</p>
+
+    <span>Yo, I'm a span tag</span>
+    <span class="deck">Yo, I'm a span tag</span>
```
[the remark this answers is not in the document]
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/selector-specificity/index.html:26]: I fix it

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/selector-specificity/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    </div>
+
+    <h2 id="content-header">Content</h2>
+
+    <div id="main-content">
+      <p>Hello there <strong>ninjas!</strong> contact us below.</p>
+      <p class="test">Aren't paragraph's cool?</p>
+    </div>
+
+    <p>None contaoned P tag</p>
```
[the remark this answers is not in the document]
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/selector-specificity/index.html]: yes, thanks

File: tasks-by-users/Naufil-Shemle/selector-specificity/style.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* this is implicitly targeting the <strong> tag */
+/* In this case main-content is the parent of <P> tag and we know that ID contains 100 points but an element contains 1 point that ID will win */
+#main-content p {
+  color: red;
+}
+
+p {
+  color: balck;
```
[the remark this answers is not in the document]
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/selector-specificity/style.css]: fixed bhai

**Comment by Naufil-Shemle** [comment]
@faran4engg  changed folder name please approve

**Review by faran4engg — APPROVED** [review]

**Pull request merged**